### DISCUSSION TITLE
Lesson 4 & 5: Ganache and pipx not recognised after being installed

### DISCUSSION THREAD

**Discussion by JamJamV** [smartcontractkit/full-blockchain-solidity-course-py#1463] [Q&A] [answered]
Hi all,
When I run the commands to install the ganache-cli & pipx, neither of them run any of the commands related to them afterwards in the tutorial. For example when I enter

> pipx install eth-brownie

I get the same message as all the other errors of 

> pipx : The term 'pipx' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
> At line:1 char:1
> + pipx install eth-brownie
> + ~~~~
>     + CategoryInfo          : ObjectNotFound: (pipx:String) [], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException

I am working w/ all my code in a GitHub repository, but I'd assume that this shouldn't affect my installation process right? (Also I think I had these issues prior to using GitHub anyways)

Thanks in advanced for any advice :)

**Comment by cromewar** [comment, marked as answer]
Hello @JamJamV 
I strongly suggest you to give a check the guide on the course index called "windows support" related to setup your environment using WSL (Windows subsystem for Linux), with that said.

1. Don't use pipx to install brownie, use regular pip instead, you should install of course python first (please do not use the windows store version) with the python oficial installer, and be sure to check install pip at the installation process.
2. Secondly, the ganache part is tricker as it depends a lot on your NodeJS installation, first of all be sure you have node correctly typing `node --version` and then try to install ganache globally.

**Reply by JamJamV** [reply]
Okay thanks, will give this a try!
